pkg/engine/harpoon.go

```diff
@@ -467,8 +467,14 @@ func (hc *HarpoonConfig) EngineMethod(ctx context.Context, path, method string,
 		return rawPodman(ctx, path, target.Raw.PullImage, prev)
 	case systemdMethod:
 		// TODO: add logic for non-root services
+		var prev *string = nil
```

Comment:
more idiomatic to not initialize the ptr since go's default ("zero") value is nil.